src/renderToStream/createPipeWrapper.ts

```diff
@@ -35,9 +35,9 @@ async function createPipeWrapper(
           writable.end()
           callback()
         },
-        // If we don't define `destroy()`, then Node.js will `process.exit()`
         destroy(err) {
-          onError(err)
+          onFatalError(err)
+          writable.destroy(err ?? undefined)
```

Comment:
Why `?? undefined`?

Reply:
`err` is `Error | null` while `writable.destroy()` accepts `Error | undefined` 🤦